Nos/Assets/Localization/Localized.swift

```diff
@@ -161,6 +161,7 @@ enum Localized: String, Localizable, CaseIterable {
     case copyLink = "Copy Link"
     case copyUserIdentifier = "Copy User ID (npub)"
     case deleteNote = "Delete"
+    case deleteNoteConfirmation = "This will delete a note that is publicly visible. Are you sure?"
```

Comment:
Probably a good place to inform the user that we cannot guarantee that the note will be effectively removed due to the nature of the decentralized web

Reply:
Yes, maybe "This will ask all your relay servers to permanently remove this note. Are you sure?"?